templates/K8cherService/.template.config/template.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json.schemastore.org/template",
```

Comment:
@turnkey-commerce I thought this might interest you. dotnet new is more flexible than I realized. It will search all files (including docker, etc.) and replace the sourceName with what is passed in the command line. This is not limited to .net.